### PR TITLE
Tweaks syndicate infiltrator and box/meta whiteships.

### PR DESCRIPTION
**Pull request by WJohn**
Box and metastation whiteships now no longer have any knockdown when you travel, but also spend 250 ticks in hyperspace instead of 10. This, coupled with not needing to be in a chair to avoid the annoying milisecond long knockdown, incentivizes people to walk around their ship and maybe fix it up or do something in it or bring some friends to go talk to when you're out adventuring all over the place.

The syndie infiltrator had its windows slightly changed to look prettier, and the northwest docking port of boxstation now works again. (The port had no width value put in).

:cl: WJohnston
tweak: Due to age, the abandoned ship's engines are now considerably slower when bringing it place to place. This does however mean that the ship now has a more gradual takeoff, with no sudden jolt to knock you off your feet.
/:cl:

### CODE REVIEW

**Comment by AnonymousNow** [comment]
That's a considerable length of time, isn't it? Does this happen for jumps within the same station Z-level?

**Comment by WJohn** [comment]
It's an increase from 10 seconds to 25 seconds, so not that bad. Metastation's whiteship has quite a lot to look around in it. I wanted it to feel more like you're on a journey from place to place, which is why I removed the knockdown so you can walk around while it's travelling without buckling yourself down the entire time in the cockpit. Less like a magic box that way.

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -14495,6 +14494,16 @@
 	},
 /turf/closed/wall/mineral/plastitanium,
 /area/shuttle/syndicate/medical)
+"ZZ" = (
+/obj/structure/grille,
+/obj/structure/window/plastitanium,
+/turf/open/floor/plating,
+/area/shuttle/syndicate/medical)
+"OVERFLOW" = (
```
Comment (ShizCalev) [inline _maps/map_files/generic/CentCom.dmm]: welp

**Comment by AnonymousNow** [comment]
If ticks are equal to seconds, more or less, then your original post says that you're upping the time to 4 1/6 minutes, not 25 seconds.

**Comment by ShizCalev** [comment]
@AnonymousNow callTime is in deciseconds.

**Comment by AnonymousNow** [comment]
So it's the ten that represents one second that's throwing me off.

**Comment by WJohn** [comment]
Apparently there's still an overflow despite me reverting and then changing more tiles when I try again.

File: _maps/shuttles/whiteship_box.dmm [hunk 2]
```diff
@@ -11,17 +11,28 @@
...
 	height = 22;
 	id = "whiteship";
 	launch_status = 0;
+	movement_force = list("KNOCKDOWN" = 0, "THROW" = 0);
 	name = "NT Medical Ship";
 	port_direction = 8;
 	preferred_direction = 4;
 	roundstart_move = "whiteship_away";
-	timid = 1;
+	timid = null;
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_box.dmm]: This will break the template version. 
timid = 1 please.

File: _maps/shuttles/whiteship_meta.dmm [hunk 3]
```diff
@@ -20,12 +20,14 @@
...
 	dheight = 0;
 	dir = 2;
 	dwidth = 11;
 	height = 15;
 	id = "whiteship";
 	launch_status = 0;
+	movement_force = list("KNOCKDOWN" = 0, "THROW" = 0);
 	name = "NT Recovery White-Ship";
 	port_direction = 8;
 	preferred_direction = 4;
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_meta.dmm]: add timid = 1; here if you wouldn't mind.

File: _maps/shuttles/whiteship_box.dmm [hunk 4]
```diff
@@ -11,17 +11,28 @@
...
+	width = 35
+	},
+/obj/docking_port/stationary{
```
Comment (ShizCalev) [inline _maps/shuttles/whiteship_box.dmm]: iirc doesn't having two docking_port modules on a template shuttle prevent it from loading?

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 5]
```diff
@@ -58902,14 +58904,27 @@
 /area/science/xenobiology)
 "Qlj" = (
 /obj/docking_port/stationary{
- 	dheight = 9;
- 	dir = 2;
- 	dwidth = 5;
- 	height = 24;
+	 dheight = 9;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Extra space?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by WJohn** [comment]
Gonna remake this PR one sec.

**Pull request closed without merging**